requirements.txt

```diff
@@ -6,7 +6,6 @@ pbr!=2.1.0,>=2.0.0,!=4.0.0,!=4.0.1,!=4.0.2,!=4.0.3 # Apache-2.0
 cliff>=2.8.0 # Apache-2.0
 python-subunit>=1.3.0 # Apache-2.0/BSD
 fixtures>=3.0.0 # Apache-2.0/BSD
-six>=1.10.0 # MIT
```

Comment:
tools/testr_to_stestr.py still imports six for `six.moves.configparser.ConfigParser()`. How about changing it to `import configparser`, instead?

```
tools/testr_to_stestr.py
18:import six
24:    config = six.moves.configparser.ConfigParser()
```

Reply:
Ah, good catch. Yeah I didn't do a grep in `tools/`. This should be fixed in: https://github.com/mtreinish/stestr/pull/287/commits/e6ce637a575eb2f58d1df58dcdc93e9b33da8307